aio/content/examples/http/src/app/heroes/heroes.component.ts

```diff
@@ -1,4 +1,4 @@
-import { Component, OnInit } from '@angular/core';
+import { ChangeDetectorRef, Component, ElementRef, OnInit, QueryList, ViewChild, ViewChildren } from '@angular/core';
```

Comment:
Some of these imports are no longer needed:

```suggestion
import { Component, ElementRef, OnInit, ViewChild } from '@angular/core';
```

Reply:
:cry: :bow:

Reply:
sorry :sob: